src/server/game/Entities/Creature/Creature.h

```diff
@@ -659,7 +659,7 @@ class Creature : public Unit, public GridObject<Creature>, public MapObject
 
         // Handling caster facing during spellcast
         void SetTarget(ObjectGuid guid) override;
-        void FocusTarget(Spell const* focusSpell, WorldObject const* target);
+        void FocusTarget(Spell const* focusSpell, ObjectGuid targetGuid);
```

Comment:
To allow focus on "nothing" with ObjectGuid::Empty
I can change it back and use ObjectGuid::Empty if target does not exist if you prefer that :)